test/Constraints/tuple_arguments.swift

```diff
@@ -1336,26 +1336,26 @@ do {
   s.takesClosureTuple({ _ = $0 })
   s.takesClosureTuple({ x in })
   s.takesClosureTuple({ (x: (Double, Double)) in })
-  s.takesClosureTuple({ _ = $0; _ = $1 }) // expected-error {{cannot convert value of type '(_, _) -> ()' to expected argument type '((Double, Double)) -> ()'}}
-  s.takesClosureTuple({ (x, y) in }) // expected-error {{cannot convert value of type '(_, _) -> ()' to expected argument type '((Double, Double)) -> ()'}}
-  s.takesClosureTuple({ (x: Double, y:Double) in }) // expected-error {{cannot convert value of type '(Double, Double) -> ()' to expected argument type '((Double, Double)) -> ()'}}
+  s.takesClosureTuple({ _ = $0; _ = $1 }) // expected-error {{closure parameter list '(Double, Double)' does not support tuple destructuring with implicit parameters}}
```

Comment:
This seems a bit odd since the parameter here is a single tuple with (Double, Double), but printing it this way after the word 'list' makes it seem like there are in fact multiple arguments here.

The same applies to other errors below.

Is it possible to make this something like "closure tuple parameter of type '(Double, Double)' cannot be destructured using implicit parameters"?